Library/Homebrew/language/python.rb

```diff
@@ -228,9 +231,33 @@ def virtualenv_install_with_resources(using: nil, system_site_packages: true, wi
           python = T.must(wanted.first)
           python = "python3" if python == "python"
         end
+
+        venv_resources = if without.blank? && start_with.blank? && end_with.blank?
+          resources
+        else
+          remaining_resources = resources.to_h { |resource| [resource.name, resource] }
+          unless (unknown_resource = Array(without).find { |r| remaining_resources.delete(r).nil? }).nil?
```

Comment:
```suggestion
          if (unknown_resource = Array(without).find { |r| remaining_resources.delete(r).nil? })
```

might be easier to follow (and same below) if it actually works as expected!